ev3dev/core.py

```diff
@@ -605,9 +616,20 @@ def time_sp(self):
     @time_sp.setter
     def time_sp(self, value):
         self.set_attr_int('time_sp', value)
+# ~autogen
 
+    def wait_for_running(self):
+        while True:
+            if 'running' in self.state:
+                return
+            sleep(0.01)
```

Comment:
> Is it possible for the state to not contain 'running' after the run command was sent to the motor?

Yes, with polled motor controllers (I2C, UART, etc.) it could take until the next poll for the running flag to be set.